### ISSUE TITLE
npx @techdocs/cli serve does not work for techdocs/cli@0.8.12

### ISSUE THREAD

**Issue by bw-bmbarga** [backstage/backstage#9424]
I used to run `npx @techdocs/cli serve` to preview techdocs locally, and since I updated to the version `0.8.12` it only shows a blank page with some errors on the console.

## Expected Behavior

When running `npx @techdocs/cli serve` a preview of the techdocs should show up on my browser

## Current Behavior

When running `npx @techdocs/cli serve` the browser only displays a blank screen with some errors on the console.

<img width="625" alt="Screenshot 2022-02-09 at 05 58 23" src="https://user-images.githubusercontent.com/39910776/153125609-774ce9ea-d3c8-4a65-bac3-f0e00862bc23.png">

```
react-dom.production.min.js:216 TypeError: Cannot read properties of undefined (reading 'next')
    at c.setActiveThemeId (AppThemeSelector.ts:79:18)
    at Function.createWithStorage (AppThemeSelector.ts:40:14)
    at AppManager.tsx:204:15
    at Object.useMemo (react-dom.production.min.js:172:115)
    at $.useMemo (react.production.min.js:23:113)
    at AppManager.tsx:203:27
    at ui (react-dom.production.min.js:157:137)
    at ws (react-dom.production.min.js:267:460)
    at hs (react-dom.production.min.js:250:347)
    at Za (react-dom.production.min.js:250:278)
```

## Possible Solution

The solution I have found so far is to run a previous version `v0.8.7` of the cli as such `npx @techdocs/cli@0.8.7 serve -v`

## Steps to Reproduce

1. Go to a folder containing an `mkdocs.yml` file with a `docs` folder 
2. Run `npx @techdocs/cli serve -v` or `npx @techdocs/cli@0.8.12 serve -v`
3. You should see a blank page on your browser and few errors on the console

## Context

I was trying to preview a basic techdoc with an `index.md` file containing the "hello world" message

## Your Environment

- NodeJS version : v16.13.0

- Operating System and Version :
Mac OS Mojave version 10.14.6
Darwin Kernel Version 18.7.0: Tue Jun 22 19:37:08 PDT 2021; root:xnu-4903.278.70~1/RELEASE_X86_64

- Browser Information:
Google chrome Version 98.0.4758.80 (Official Build) (x86_64)

**Comment by Rugvip** [comment]
This should be fixed by #9370. It'll be available in the release tomorrow, but you could already try it out with `@techdocs/cli@0.8.13-next.0`

**Comment by Rugvip** [comment]
Gonna close this as the fix has now been released. Let us know if you still see the issue when using `0.8.13` or above
